### PR TITLE
app-admin/logstash-bin: bump to 5.1.1

### PR DESCRIPTION
**Pull request by hydrapolic**
Package-Manager: portage-2.3.3

### CODE REVIEW

**Comment by gktrk** [comment]
Can you revision the sample file (i.e. sample-r1). Also:

```
--- logstash-bin-5.0.1-r1.ebuild	2016-12-18 11:26:16.114718356 -0500
+++ logstash-bin-5.1.1.ebuild	2016-12-21 21:43:06.596506536 -0500
@@ -36,7 +36,7 @@
 	keepdir "/var/log/${MY_PN}"
 
 	insinto "/usr/share/${MY_PN}"
-	doins "${FILESDIR}/agent.conf.sample"
+	newins "${FILESDIR}/agent.conf.sample2" agent.conf
 
 	insinto "/opt/${MY_PN}"
 	doins -r .
```

Why did we start overwriting `agent.conf` all of a sudden instead of providing a sample file?

File: app-admin/logstash-bin/logstash-bin-5.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	newins "${FILESDIR}/${MY_PN}.logrotate" "${MY_PN}"
+
+	newconfd "${FILESDIR}/${MY_PN}.confd2" "${MY_PN}"
+	newinitd "${FILESDIR}/${MY_PN}.initd4" "${MY_PN}"
+}
+
+pkg_postinst() {
+	ewarn "The default user changed from root to ${MY_PN}. If you wish to run as root (for"
+	ewarn "example to read local logs), be sure to change LS_USER and LS_GROUP in"
+	ewarn "/etc/conf.d/${MY_PN}"
```
Comment (gktrk) [inline app-admin/logstash-bin/logstash-bin-5.1.1.ebuild]: `${EROOT%/}/etc/conf.d/${MY_PN}`

**Comment by hydrapolic** [comment]
@gktrk, we provide just one sample file. Since the ES output syntax changed with versions 5.x, I modified the sample file. The old syntax seems to work, but is deprecated and not listed in the docs.

Thanks for the notice about EROOT, I'll try to modify and push when my children are asleep :)

**Comment by hydrapolic** [comment]
@gktrk, done.

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/47188a6...b7d73b1. Thanks!

**Comment by hydrapolic** [comment]
Thank you

**Pull request closed without merging**